### PR TITLE
Using the newer gem structure & Added sidekiq health check rake task

### PR DESCRIPTION
**Pull request by kmpzr**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```
Comment (TheWudu) [inline Gemfile]: needed?

File: health_monitor.gemspec [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+  spec.homepage      = "https://github.com/runtastic/health_monitor"
+
+  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
+    f.match(%r{^(test|spec|features)/})
+  end
+  spec.bindir        = "exe"
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ["lib"]
+
+  spec.add_development_dependency "bundler", "~> 1.16"
```
Comment (TheWudu) [inline health_monitor.gemspec]: does it need a version lock?

File: lib/health_monitor/rake_task.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      hostname = ENV["HOSTNAME"] || `hostname -f`.strip
+      ps = Sidekiq::ProcessSet.new
+      up = ps.any? { |p| p["hostname"] == hostname && p["quiet"] == "false" }
+
+      if up
+        puts "UP"
+        exit 0
+      end
+
+      puts "DOWN"
```
Comment (TheWudu) [inline lib/health_monitor/rake_task.rb:20]: the puts are necessary for OS ?

Comment (kmpzr) [reply lib/health_monitor/rake_task.rb:20]: This is better for Debugging, if we want to investigate if there is something happening with the script inside an OS POD. The OS Shell acts weird sometimes. It is also good if there is any logging on the Health Check, we could see what and when it happened in our Logging view interface.

File: Gemfile (hunk 1 above)
Comment (kmpzr) [reply Gemfile]: Not needed, just standard bundler gem generation garbage. I'll remove.

File: health_monitor.gemspec (hunk 2 above)
Comment (kmpzr) [reply health_monitor.gemspec]: Not needed, just standard bundler gem generation garbage. I'll remove this and the other locks as well.

**Review by TheWudu — APPROVED** [review]

**Pull request merged**